my-app/src/components/register/register.js

```diff
@@ -1,18 +1,41 @@
 import React from 'react';
 import './register.css'; 
-import {Link} from "react-router-dom";
+//import {Link} from "react-router-dom";
+import {Link, withRouter} from "react-router-dom";
+
+function validate(username, password, first, last) {
+  const errors = [];
+
+  if (username.length <= 4) {
```

Comment:
Many valid usernames such as sam or Omar are unavailable because of this, maybe consider changing the value to something less like 1 or 2?

Reply:
Thanks for letting me know, I made some changes.